server/model/model.go

```diff
@@ -139,7 +128,18 @@ type SearchNeed struct {
 	CategorySlug      string       `db:"category_slug"`
 }
 
-func (s *needStatus) Scan(src interface{}) error {
+type NeedStatus string
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type NeedStatus should have comment or be unexported](https://app.codacy.com/app/coderockrdev/vitrine-social/pullRequest?prid=1995521)